attributes/default.rb

```diff
@@ -79,7 +79,19 @@
 default["kafka"]["offset_monitor"]["refresh"] = "15.minutes"
 default["kafka"]["offset_monitor"]["retain"] = "7.days"
 
+default["kafka"]["offset_monitor"]["include_log4j_jar"] = true
```

Comment:
Missed this one added it to the README, https://github.com/cerner/cerner_kafka/pull/52/commits/5b5c303a750baa89884be467aed3358cc60fde4e